.github/actions/prepare-clojure-env/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Prepare clojure env'
+description: 'Prepare clojure environment for build'
+runs:
+  using: "composite"
+  steps:
+    - name: Cache local Maven repository
+      uses: actions/cache@v3
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/project.clj') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Install clojure tools
+      uses: DeLaGuardo/setup-clojure@9.5
```

Comment:
Tä taitaa vastata travis.yml:n `language: clojure` asetusta?

Reply:
Joo, jossain määrin näin. Mitään ihan vastaavaa ei github actionsin puolelta löydy, mutta tää asentaa `lein`in käyttöön, ja se ainakin tarvitaan.